pipeline_dp/dp_computations.py

```diff
@@ -641,3 +643,42 @@ def create_additive_mechanism(
         return GaussianMechanism(spec.epsilon, spec.delta, sensitivities.l2)
 
     assert False, f"{spec.noise_kind} not supported."
+
+
+class ExponentialMechanism:
+
+    class ScoringFunction(abc.ABC):
+
+        def __init__(self) -> None:
+            super().__init__()
+
+        @abc.abstractmethod
+        def score(self, k) -> float:
+            """"""
```

Comment:
Nit: pass or (even better) extend docstring

Reply:
what do you mean?

Reply:
it's better 
```
def score():
  pass
```
or
```
def score():
  """docstring"""
```